traffic_portal/app/src/common/modules/table/servers/TableServersController.js

```diff
@@ -23,13 +23,22 @@ var TableServersController = function(tableName, servers, filter, $scope, $state
 	// browserify can't handle classes...
 	function SSHCellRenderer() {}
 	SSHCellRenderer.prototype.init = function(params) {
-		this.eGui = document.createElement("A");
-		this.eGui.href = "ssh://" + userModel.user.username + "@" + params.value;
-		this.eGui.setAttribute("target", "_blank");
+		this.eGui = document.createElement("div");
 		this.eGui.textContent = params.value;
+		this.data.cellHrefValue = "ssh://" + userModel.user.username + "@" + params.value;
+
 	};
 	SSHCellRenderer.prototype.getGui = function() {return this.eGui;};
 
+	function ILOCellRenderer() {}
+	ILOCellRenderer.prototype.init = function(params) {
+		this.eGui = document.createElement("div");
```

Comment:
ILOCellRenderer never gets initialized. I believe it's because all of the columns it's on are hidden by default (the SSHCellRenderer has one that is not, so it does get initialized).

Reply:
:thinking: I'll dig into this one tomorrow